### ISSUE TITLE
Loinc 1992-7:  "Calcitonin [Mass/volume] in Serum or Plasma"

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#4104]
Suggest creating a new term for Loinc: 1992-7 [Calcitonin [Mass/volume] in Serum or Plasma] 
New term label: Decreased calcitonin
Parent term: Abnormality of calcium-phosphate metabolism
New term comment (if any): see: https://labtestsonline.org/tests/calcitonin
Your biocurator ID for loinc2hpo (if desired): JGM:lccarmody

**Comment by pnrobinson** [comment]
I do not think that decreased calcitonin is medically relevant.
We can stick with
https://hpo.jax.org/app/browse/term/HP:0003528
